deepchem/transformers/__init__.py

```diff
@@ -95,7 +100,7 @@ def _transform_row(i, df, transformer, data_dir):
 class NormalizationTransformer(Transformer):
 
   def __init__(self, transform_X=False, transform_y=False, transform_w=False,
-               dataset=None):
+               dataset=None, transform_gradients=False):
```

Comment:
@rbharath Standard Normalization transformer is used for option 1 and 3 where we are not training on tf.gradient(E).  `untransform_grad` is used in the predict grad function of option 3.  I had two different version of NormalizationTransform because this one was fine for options 1 and 3, but option two needed the gradient to be properly transformed during model.fit()